packages/teams-js/src/public/sharing.ts

```diff
@@ -194,4 +194,80 @@ export namespace sharing {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.sharing ? true : false;
   }
+
+  /**
+   * Namespace to get the list of content shared in a context
+   *
+   * @beta
+   */
+  export namespace history {
+    /**
+     * Represents IContentResponse parameters.
+     *
+     * @beta
+     */
+    export interface IContentResponse {
+      /** Id of the app where the content was shared from */
+      appId: string;
+      /** Title of the shared content */
+      title: string;
+      /** Reference of the shared content */
+      contentReference: string;
+      /** Id of the thread where the content was shared */
+      threadId: string;
+      /** Id of the user who shared the content */
+      author: string;
+      /** Type of the shared content */
+      contentType: string;
+    }
+
+    /**
+     * getContent callback function type
+     *
+     * @beta
+     */
+    export type getContentCallbackFunctionType = (
+      error: SdkError | null,
+      contentDetails: IContentResponse[] | null,
+    ) => void;
+
+    /**
+     * Get the list of content shared in a context
```

Comment:
>shared in a context [](http://example.com/codeflow?start=31&length=19)

Can you elaborate on what "shared in a context" means? I'm not sure that developers reading this would know how to interpret that phrase.

Reply:
Updated comment